planner/core/memtable_predicate_extractor.go

```diff
@@ -528,6 +532,10 @@ func (e *ClusterTableExtractor) Extract(_ sessionctx.Context,
 	return remained
 }
 
+func (e *ClusterTableExtractor) explainInfo(p *PhysicalMemTable) string {
```

Comment:
Prefer to add a `dummyExplainInfo` instead of adding empty implementation for all extractor.

Reply:
No,becase UCP https://github.com/pingcap/tidb/issues/15378 need to implement those methods.